ev3dev/core.py

```diff
@@ -955,6 +958,94 @@ def wait_while(self, s, timeout=None):
         """
         return self.wait(lambda state: s not in state, timeout)
 
+    def set_power(self, power):
+        self.speed_sp = power
+
+    def set_position_rotations(self, power, rotations):
+        if power > 0:
+            self.position_sp = self.position + int(rotations * self.count_per_rot)
+        else:
+            self.position_sp = self.position - int(rotations * self.count_per_rot)
+
+    def set_position_degrees(self, power, degrees):
+        rotations = float(degrees / self.count_per_rot)
```

Comment:
This codepath will read from `count_per_rot` twice. Can we restructure this to avoid that?

Reply:
I could cache it here or we could do something in the Device class (or maybe the Motor class) where it is aware of what properties will never change and cache it there.  If we want to do the latter I'll work on that as a separate PR.

Reply:
RE: properties that never change. See:

* https://github.com/ev3go/ev3dev/issues/53
* https://github.com/ev3go/ev3dev/issues/54
* https://github.com/ev3go/ev3dev/issues/55

Reply:
Filed issued #365 to track this